nomenclature/testing.py

```diff
@@ -2,16 +2,38 @@
 import logging
 from pathlib import Path
 
+import click
+
+import nomenclature
 
 logger = logging.getLogger(__name__)
 
+cli = click.Group()
+
+
+@cli.command("validate-yaml")
+@click.argument(
+    "path", type=click.Path(exists=True, path_type=Path, file_okay=False, dir_okay=True)
+)
+def cli_valid_yaml(path: Path):
+    assert_valid_yaml(path)
+
+
+@cli.command("validate-project")
+@click.argument(
+    "path", type=click.Path(exists=True, path_type=Path, file_okay=False, dir_okay=True)
```

Comment:
Seems that the args `file_okay` and `dir_okay` are not used? Also, maybe use `check_file` and `check_dir` instead (if these arguments are needed)...?

Reply:
I guess at this point it is okay to leave out the arguments?